### ISSUE TITLE
FontDescription.LoadDescription throws ArgumentException

### ISSUE THREAD

**Issue by Metalnem** [SixLabors/Fonts#96]
**FontDescription.LoadDescription** throws an **ArgumentException** when loading the font from the [attached archive]. You can run the following code to reproduce it (the **path** variable should contain the path to the extracted file):

```csharp
FontDescription.LoadDescription(path);
```

Found via [SharpFuzz].

[attached archive]: https://github.com/SixLabors/Fonts/files/2707142/ArgumentException.zip
[SharpFuzz]: https://github.com/Metalnem/sharpfuzz

**Comment by dlemstra** [comment]
The full exception is:

```
System.ArgumentException : An item with the same key has already been added. Key: kern
Stack Trace:
  at System.Collections.Generic.Dictionary`2.TryInsert(TKey key, TValue value, InsertionBehavior behavior)
  at System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
  at SixLabors.Fonts.FontReader..ctor(Stream stream, TableLoader loader)
  at SixLabors.Fonts.FontReader..ctor(Stream stream)
  at SixLabors.Fonts.FontDescription.LoadDescription(String path)
```

Okay if I add a patch that does `headers[tbl.Tag]` instead of `headers.Add(tbl.Tag)` @tocsoft?

**Comment by tocsoft** [comment]
no real problem, not sure its a valid font file with duplicate tables but preventing it breaking with an unhelpful error is better

**Comment by Metalnem** [comment]
Not a valid font - it was obtained by fuzzing the library. The fuzzer started with a small valid font and generated thousands of random mutations before generating this particular one.

**Comment by dlemstra** [comment]
What would you prefer that we do if we find a duplicate table? Don't understand your answer 😉

**Comment by tocsoft** [comment]
Just replace the previous entry for now like you suggested. We can worry about correctness later as I can't currently find anything saying it's invalid to have duplicate tables defined.
